### PR TITLE
Update IDL with performance timeline L2

### PR DESCRIPTION
**Pull request by plehegar**

### CODE REVIEW

**Comment by w3c-bots** [comment]
# Build PENDING

Started: None
Finished: None





View more information about this build on:

- [WPT PR Status](http://pulls.web-platform-tests.org/build/18864)
- [TravisCI](https://travis-ci.org/w3c/web-platform-tests/builds/279979920)

**Review by igrigorik — APPROVED** [review]
👍

File: performance-timeline/idlharness.html [hunk 1]
```diff
@@ -75,8 +76,11 @@
   idl_array = new IdlArray();
   idl_array.add_untested_idls(document.getElementById("untested_idl").textContent);
   idl_array.add_idls(document.getElementById("idl").textContent);
+  // create first mark
+  self.performance.mark("mark");
   idl_array.add_objects({
-    Performance: ["window.performance"]
+    Performance: ["window.performance"],
+    PerformanceMark: [self.performance.getEntriesByName("mark")[0]]
```
Comment (toddreifsteck) [inline performance-timeline/idlharness.html]: Why are we testing "mark"? So we can instantiate a PerformanceEntry?

**Review by toddreifsteck — COMMENTED** [review]
Ensuring that @igrigorik  and I 100% understand these tests.

File: performance-timeline/idlharness.html (hunk 1 above)
Comment (plehegar) [reply performance-timeline/idlharness.html]: yes indeed. I need to instanciate one PerformanceEntry to trigger the toJSON test. I figured mark was the minimal one I could pick.

**Pull request merged**